pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -134,6 +136,13 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 		} else {
 			return nil, maxMonID, monMapping, errors.New("failed to find either the cluster admin key or the username")
 		}
+		if clusterInfo.Spec.Annotations != nil {
```

Comment:
As discussed in huddle, maybe ok to just remove this check and always update the annotations in case they were removed.

Reply:
removed